pyInterface/userInterface/eigenvectorLikelihoodSlices.py

```diff
@@ -78,12 +79,13 @@
 	covMatrixMinuit = result.fitParCovMatrix()
 	eigenVectorsMinuit = likelihood.HessianEigenVectors(covMatrixMinuit)
 
-	covMatrixAna = likelihood.CovarianceMatrix( [ minimum[i] for i in xrange(minimum.GetNrows()) ] )
-	eigenVectorsAna = likelihood.HessianEigenVectors(covMatrixAna)
+	if not args.ranWithHesse:
+		covMatrixAna = likelihood.CovarianceMatrix( [ minimum[i] for i in xrange(minimum.GetNrows()) ] )
+		eigenVectorsAna = likelihood.HessianEigenVectors(covMatrixAna)
 
-	if len(eigenVectorsMinuit) != len(eigenVectorsMinuit):
-		pyRootPwa.utils.printErr("different number of Eigenvalues for covariance matrix of Minuit vs. analytic covariance matrix.")
-		sys.exit(1)
+		if len(eigenVectorsMinuit) != len(eigenVectorsMinuit):
```

Comment:
This comparison should be fixed, we should comare `eigenVectorsMinuit` and `eigenVectorsAna` here. Don't know why I didn't write this correctly in the first place, ...